### PR TITLE
T32077 cards

### PR DESCRIPTION
**Pull request by manuq**
https://phabricator.endlessm.com/T32077

### CODE REVIEW

File: packages/eos-components/src/components/mixins/cardMixin.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default {
   },
   methods: {
     getThumbnail() {
-      if (!this.node.thumbnail && process.env.VUE_APP_USE_MOCK_DATA === 'true') {
+      if (!this.node.thumbnail || this.node.useDefaultThumbnail) {
```
Comment (danigm) [inline packages/eos-components/src/components/mixins/cardMixin.js:26]: :+1:  Good idea to clean this kind of code that we're not using anymore.

File: packages/eos-components/src/utils.js [hunk 2]
```diff
@@ -68,7 +68,11 @@ export function getNodeUrl(node, channelId) {
 
 export function getLeaves(node) {
   if (!node.children) {
-    return [node];
+    if (node.kind === 'topic') {
+      return [];
+    } else {
```
Comment (danigm) [inline packages/eos-components/src/utils.js:73]: This else is not needed after a return: https://eslint.org/docs/rules/no-else-return

**Review by danigm — APPROVED** [review]
Looks good, just one eslint problem to fix.

File: packages/eos-components/src/utils.js (hunk 2 above)
Comment (manuq) [reply packages/eos-components/src/utils.js:73]: OK! I can change this, I prefer that too. But that eslint rule is not complaining in our checks.

**Pull request merged**